<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcompetitiondlg</class>
 <widget class="QDialog" name="addcompetitiondlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>203</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>竞赛代码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idLineEdit">
        <property name="placeholderText">
         <string>此项必填</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>竞赛名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLineEdit_2">
        <property name="placeholderText">
         <string>此项必填</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>竞赛级别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="degreeComboBox">
        <item>
         <property name="text">
          <string>院级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>校级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>省级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>国家级</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>竞赛类别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="qualityComboBox_2">
        <item>
         <property name="text">
          <string>红色赛道</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>本科创新</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>研究生组</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
